webapp/actions/post_actions.jsx

```diff
@@ -272,3 +272,38 @@ export function removeReaction(channelId, postId, emojiName) {
 
     AsyncClient.deleteReaction(channelId, reaction);
 }
+
+export function createPost(post, doLoadPost, success, error) {
+    Client.createPost(post,
+        (data) => {
+            if (doLoadPost) {
+                loadPosts(post.channel_id);
+            } else {
+                PostStore.removePendingPost(post.pending_post_id);
+            }
+
+            AppDispatcher.handleServerAction({
+                type: ActionTypes.RECEIVED_POST,
+                post: data
+            });
+
+            if (success) {
+                success(data);
+            }
+        },
+
+        (err) => {
+            if (err.id === 'api.post.create_post.root_id.app_error') {
+                // this should never actually happen since you can't reply from this textbox
```

Comment:
Sorry @cpanato, just one more thing. Can you remove this comment? I don't think it makes sense in this context